PHP/Embeddings/SQL (for PHP Interpolated).sublime-syntax

```diff
@@ -34,3 +34,8 @@ contexts:
     # this context is included in anonymous string contexts
     - meta_prepend: true
     - include: Packages/PHP/PHP Source.sublime-syntax#string-interpolations
+
+  table-name-or-subquery:
+    - meta_include_prototype: false
+    - meta_prepend: true
+    - include: comments
```

Comment:
In case `meta_include_prototype` is required to correctly handle interpolation, we should probably (keep) it in core SQL syntax. Otherwise extending syntaxes need to add too many special-case contesxts like this one, increasing dependency surface.

Reply:
the `meta_include_prototype` for `table-name-or-subquery`  is now back in the core/basic SQL syntax :+1: